packages/composite-checkout/src/components/checkout-order-summary.tsx

```diff
@@ -1,9 +1,9 @@
+import styled from '@emotion/styled';
 import { useI18n } from '@wordpress/react-i18n';
 import React from 'react';
-import styled from '../lib/styled';
 import { CheckoutSummaryCard, useLineItems, useLineItemsOfType, useTotal } from '../public-api';
 
-export default function CheckoutOrderSummaryStep() {
+export default function CheckoutOrderSummaryStep(): JSX.Element {
```

Comment:
Maybe a better type would be [`React.FunctionComponent`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L543) ?

(this comment would apply to every other instance of React's functional component types with this pattern)

Reply:
IMO using `JSX.Element` is nicer because you can use the `export function` syntax and don't need to do `const` with an arrow function and then `export const`.